src/StableTrieMap.mo

```diff
@@ -146,23 +146,25 @@ module {
       Region.loadNat64(region.region, getOffset(node, index)) & loadMask;
     };
 
+    let storePointer : (region : Region.Region, offset : Nat64, child : Nat64) -> () = switch (pointer_size_) {
```

Comment:
We can remove the `region` argument from both `storePointer` and `setChild`. Both functions are only ever called with `region == nodes`.